app/AndroidManifest.xml

```diff
@@ -2,8 +2,8 @@
 <manifest package="org.commcare.dalvik"
           xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
-          android:versionCode="106"
-          android:versionName="2.52">
+          android:versionCode="107"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/58fdac3d3be16630817ec8382851f037db49d9c2/app/AndroidManifest.xml#L5"></span>This <code>versionCode</code> value (<code>107</code>) is not used; it is always overridden by the value specified in the Gradle build script (<code>1</code>)</td>
    </tr>
  </tbody>
</table>